<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1356</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Пахомов ДВ 224-321</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>211</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Параметры ГА</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Размер популяции</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Size_population_spinBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Кол-во поколений</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Count_General_spinBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Вероятность мутации</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="Mutantion_doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="Start_GA_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Запуск ГА</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>531</height>
     </rect>
    </property>
    <property name="title">
     <string>Параметры графа</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Кол-во точек</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Count_points_spinBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QPushButton" name="Generate_graph_pushButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Сгенерировать топологию</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Save_Graph_pushButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Start_point_spinBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Начало</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Finish_points_spinBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Конец</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="Adjacency_TabletableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>451</width>
       <height>441</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Back_Graph_pushButton">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>40</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="Log_textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>691</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>641</width>
      <height>661</height>
     </rect>
    </property>
    <property name="title">
     <string>Отображение</string>
    </property>
    <widget class="QLabel" name="Image_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>600</width>
       <height>600</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Graph_view_checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Граф</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="Statistics_view_checkBox_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Статистика</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
